db/migrations/0011.add_bluejeans_table.py

```diff
@@ -0,0 +1,42 @@
+#
+# file: migrations/0011.add_bluejeans_table.py
+#
+from yoyo import step
+
+__depends__ = {'0010.add_fkc_course_usage_table'}
+steps = [
+    step('''
+        CREATE TABLE IF NOT EXISTS bluejeans_meetings
+        (
+            meeting_uuid varchar(255) NOT NULL UNIQUE,
+            meetingTitle VARCHAR(255) NULL,
+            meetingId varchar(50) NOT NULL,
+            userName VARCHAR(255) NOT NULL,
+            user_id VARCHAR(255) NOT NULL,
+            user_tags  VARCHAR(255),
+            startTime DATETIME,
+            endTime DATETIME,
+            email VARCHAR(255),
+            participantSeconds int,
+            joinDate DATETIME,
```

Comment:
Can you be consistent with the table naming convention? Here is a quick guideline https://launchbylunch.com/posts/2014/Feb/16/sql-naming-conventions/#naming-conventions

Reply:
> @zqian can you resolve conflicts?

Just did!

Reply:
The table column name should be in format "lower_case_with_underscores"

Reply:
Agree with @pushyamig here, half the names are in CamelCase and the other half aren't; the rest of the database uses the underscore appraoch. Also, it would be nice to be consistent with putting data types in all caps and having one space between the column name and the data type.